src/frontend/screens/WebView/index.tsx

```diff
@@ -217,8 +217,19 @@ export default function WebView({ store }: Props) {
         }
       }
 
-      webview.addEventListener('dom-ready', loadstop)
+      const onerror = ({ validatedURL }: Electron.DidFailLoadEvent) => {
+        // Connection refused
+        if (validatedURL && validatedURL.match(/track\.adtraction\.com/)) {
+          const parsedUrl = new URL(validatedURL)
+          const redirectUrl = parsedUrl.searchParams.get('url')
+          if (redirectUrl) {
+            webview.loadURL(redirectUrl)
+          }
```

Comment:
should we add something like:

```
else {
  webview.loadURL("gog.com url here")
}
```

I understand there's a chance that `redirectUrl` can be empty/undefined and it's probably better to fallback to something that works instead of staying in the the adtraction page that failed

(I think in practice that shouldn't happen? but since the if is there I think we should handle the else for completeness)